packages/block-editor/src/components/block-alignment-control/ui.js

```diff
@@ -64,7 +64,7 @@ function BlockAlignmentUI( {
 				} ),
 		  }
 		: {
-				toggleProps: { describedBy: __( 'Change alignment' ) },
+				toggleProps: { description: __( 'Change alignment' ) },
```

Comment:
I see we've not updated the prop to the new one everywhere. Example:

https://github.com/WordPress/gutenberg/blob/0221522f253e094b277a1485b7a2d186cb172632/packages/block-editor/src/components/alignment-control/ui.js#L59

Is that intended?

Reply:
Glad to update more occurrences if desired. I think I only searched for occurrences of `\tdescribedBy=` and `\tdescribedBy:` 😆

Reply:
I think that will make sense since otherwise folks might get unexpected deprecation warnings in core.